src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/LanguageServerResources.resx

```diff
@@ -180,4 +183,13 @@
   <data name="Test_run_error" xml:space="preserve">
     <value>Test run error: {0}</value>
   </data>
+  <data name="There_were_errors_loading_project_0_See_log_for_details" xml:space="preserve">
+    <value>There were errors loading project {0}. See log for details.</value>
```

Comment:
If you want to reduce duplication, consider "problems" instead of errors/warnings; I'm not sure the person seeing the toast will particularly care on the arbitrary severity we chose.

Reply:
great suggestion, will do that.